usr/lib/common/host_defs.h

```diff
@@ -237,7 +237,7 @@ typedef struct _ATTRIBUTE_PARSE_LIST {
 typedef struct _OP_STATE_DATA {
     CK_CHAR library_version[16]; /* zero termination does not matter here */
     CK_CHAR manufacturerID[member_size(CK_TOKEN_INFO_32, manufacturerID)];
-    CK_CHAR model[member_size(CK_TOKEN_INFO_32, manufacturerID)];
+    CK_CHAR model[member_size(CK_TOKEN_INFO_32, model)];
     CK_STATE session_state;
```

Comment:
Yes, it's a typical copy/paste mistake. We just wonder why it is coded so complicated: we could simply write sizeof() instead of using this member_size() function.

Reply:
You can't do `sizeof()` of a member of a struct. Only if you have a pointer to that struct, then you could do `sizeof(ptr->member)`.... 
If you look what `member_size` does: It uses a zero-pointer casted to the struct to address the member and uses a sizeof that way.